src/file_pipe_log.rs

```diff
@@ -470,18 +496,9 @@ impl PipeLog for FilePipeLog {
     ) -> Result<()> {
         let content = self.read_file_bytes(queue, file_id)?;
```

Comment:
Leave a TODO in `fn read_file_bytes` to use `LogFd` for reading file bytes instead of std::fs?